v20.2/disaster-recovery.md

```diff
@@ -0,0 +1,363 @@
+
+## Data failure
+
+When dealing with data failure due to bad actors, rogue applications, or data corruption, domain expertise is required to identify the affected rows and determine how to to remedy the situation (e.g., remove the incorrectly inserted rows, insert deleted rows, etc.). However, there are a few actions that you can take for short-term remediation:
+
+- If you are within the garbage collection window, [run differentials](#run-differentials).
+- If you have a backup file, [restore to a point in time](#restore-to-a-point-in-time).
+- If your cluster is running and you do not have a backup with the data you need, [create a new backup](#create-a-new-backup).
+- To [recover from corrupted data in a database or table](#recover-from-corrupted-data-in-a-database-or-table), restore the corrupted object.
+
+{{site.data.alerts.callout_success}}
+To give yourself more time to recover and clean up the corrupted data, put your application in “read only” mode and only run [`AS OF SYSTEM TIME`](as-of-system-time.html) queries from the application.
+{{site.data.alerts.end}}
+
+### Run differentials
+
+If you are within the [garbage collection window](configure-replication-zones.html#replication-zone-variables) (default is 25 hours), run [`AS OF SYSTEM TIME`](as-of-system-time.html) queries and use [`CREATE TABLE AS … SELECT * FROM`](create-table-as.html) to create comparison data and run differentials to find the offending rows to fix.
+
+If you are outside of the garbage collection window, you will need to use a [backup](backup.html) to run comparisons.
+
+### Restore to a point in time
+
+- If you are a core user, use a [backup](backup.html) that was taken with [`AS OF SYSTEM TIME`](as-of-system-time.html) to restore to a specific point.
+- If you are an enterprise user, use your [backup](backup.html) file to [restore to a point in time](backup-and-restore-advanced-options.html#point-in-time-restore) where you are certain there was no corruption. Note that the backup must have been taken with [revision history](backup.html#with-revision-history).
+
+### Create a new backup
+
+If your cluster is running, you do not have a backup that encapsulates the time you want to [restore](restore.html) to, and the data you want to recover is still in the [garbage collection window](configure-replication-zones.html#replication-zone-variables), there are two actions you can take:
+
+- If you are a core user, trigger a [backup](backup.html) using [`AS OF SYSTEM TIME`](as-of-system-time.html) to create a new backup that encapsulates the specific time. The `AS OF SYSTEM TIME` must be within the [garbage collection window](configure-replication-zones.html#replication-zone-variables) (default is 25 hours).
+- If you are an enterprise user, trigger a new [backup `with_revision_history`](backup-and-restore-advanced-options.html#backup-with-revision-history-and-point-in-time-restore) and you will have a backup you can use to restore to the desired point in time.
```

Comment:
minor nit -- for enterprise user it's also `to the desired point in time within the garbage collection window`

Reply:
Done!